parl/remote/worker.py

```diff
@@ -95,14 +101,15 @@ def _set_cpu_num(self, cpu_num=None):
             self.cpu_num = multiprocessing.cpu_count()
 
     def _create_sockets(self):
-        """ Each worker has three sockets at start:
+        """ Each worker has four sockets at start:
 
         (1) request_master_socket: sends job address to master node.
         (2) reply_master_socket: accepts submitted job from master node.
         (3) reply_job_socket: receives job_address from subprocess.
+        (4) kill_job_socket : receives commands to kill the job from jobs.
 
-        When a job is start, a new heartbeat socket is created to receive
-        heartbeat signal from the job.
+        When a job starts, a new heartbeat socket is created to receive
+        heartbeat signals from the job.
 
         """
 
```

Comment:
`self.reply_master_socket` on line `125` is not used in this version.

It was used to receive `KILL_JOB_TAG` from the master when a client is done, but in this version worker will receive `KILL_JOB_TAG` with `kill_job_socket` from the job.

And the `reply_master_address` is only used as the unique `worker_address` which can be replaced by other address like `self.master_heartbeat_address`

Reply:
Can we fix this in next PR as removing this socket and address links to a number of changes on other objects, such as `worker_status`, `initlialized_jobs` and `Master`.